### PR TITLE
Pin go-msgpack to a version compatible with Consul

### PR DESCRIPTION
**Pull request by kisunji**

### CODE REVIEW

File: go.mod
Comment (kisunji) [reply go.mod:37]: No more need for this comment; v1.1.5 has been retracted

**Comment by dgregoire** [comment]
Closing this.

**Review by lmorton-tran — APPROVED** [review]
Thanks! Please wait for approval from @dgregoire first.

**Comment by kisunji** [comment]
Just closing out of precation; @dgregoire is looking into removing the dependency since `v0.5.5` is not compatible with scada brokers

**Comment by dgregoire** [comment]
> Just closing out of precaution; @dgregoire is looking into removing the dependency since `v0.5.5` is not compatible with scada brokers

False negative test, the problem is elsewhere. Interop looks ok.

**Pull request closed without merging**
